src/util/apiCalls.js

```diff
@@ -27,4 +27,21 @@ export const getInterviews = async () => {
       questions: []
     }
   ]
+}
+
+export const getQuestions = async () => {
+  return [
+    {
+      id: 1,
+      body: 'What is France?',
+      score: 4,
+      notes: 'Great answer'
+    },
+    {
+      id: 2,
+      body: 'What is life?',
+      score: 2,
+      notes: 'Bad answer'
+    }
+  ]
```

Comment:
Keep in mind that a Note is a separate object/component/model/whatever, with it's own id and body and score, and a question has an array of Notes.

Reply:
Also, we do have a query for this on the backend already.

Reply:
Sure, I was just mocking data so that I could have something to work with to make appear on the DOM.